Extensions/postarchive.js

```diff
@@ -75,6 +88,16 @@ XKit.extensions.postarchive = {
 
 		this.running = true;
 
+		window.addEventListener("message", function(e) {
+			if (e.origin !== "https://www.tumblr.com" && !e.data.xkit) { return; }
+			if (e.data.extension_method) {
+				XKit.extensions.postarchive[e.data.extension_method].apply(null, e.data.args);
+			}
+			if (e.data.expand_in_blog_control_iframe) {
+				$('.tmblr-iframe--desktop-loggedin-controls').width('100%');
+			}
+		});
```

Comment:
would prefer that expand_in_blog_control_iframe ends up in patches because its replacing a function that was in patches and depended on by other extensions, but down to punt for now (but only until someone gets around to fixing Mute or BlogFinder and needs it)